privacyidea/lib/auditmodules/loggeraudit.py

```diff
@@ -69,8 +69,10 @@ class Audit(AuditBase):
     .. note:: This audit module does not provide a *Read* capability.
     """
 
-    def __init__(self, config=None):
-        super(Audit, self).__init__(config)
+    def __init__(self, config=None, startdate=None):
+        super(Audit, self).__init__(config, startdate)
+        # overwrite the startdate from the superclass with the isoformat
+        self.audit_data["startdate"] = startdate or datetime.utcnow().isoformat()
```

Comment:
And i just realized that the duration isn't determined in the logger audit. It should be calculated in `finalize_log()`.

Reply:
Yes, I saw this before and actually I forgot about it again.
Do we want it?
OK, I will add it.

Reply:
Done in 1da2c0a5